src/internal/middleware/auth/interceptor.go

```diff
@@ -77,6 +77,9 @@ var authHandlers = map[string]authHandler{
 	"/enterprise_v2.API/GetActivationCode": authDisabledOr(clusterPermissions(auth.Permission_CLUSTER_ENTERPRISE_GET_CODE)),
 	"/enterprise_v2.API/Deactivate":        authDisabledOr(clusterPermissions(auth.Permission_CLUSTER_ENTERPRISE_DEACTIVATE)),
 	"/enterprise_v2.API/Heartbeat":         authDisabledOr(clusterPermissions(auth.Permission_CLUSTER_ENTERPRISE_HEARTBEAT)),
+	"/enterprise_v2.API/Pause":             authDisabledOr(clusterPermissions(auth.Permission_CLUSTER_ENTERPRISE_PAUSE)),
+	"/enterprise_v2.API/Unpause":           authDisabledOr(clusterPermissions(auth.Permission_CLUSTER_ENTERPRISE_PAUSE)),
+	"/enterprise_v2.API/PauseStatus":       authDisabledOr(clusterPermissions(auth.Permission_CLUSTER_ENTERPRISE_PAUSE)),
```

Comment:
Instead of creating another permission here, you can just set the value in the map to `authenticated` which will allow any cluster user to run the PauseStatus API